add-on/panels/js/sidebar.jsx

```diff
@@ -156,6 +156,9 @@ loop.sidebar = (function(mozL10n) {
         dataDriver: dataDriver
       });
 
+      dispatcher.dispatch(
+        new sharedActions.SetOwnDisplayName({ displayName: "Wanda RoomOwner" }));
```

Comment:
Should this be defaulting or just using the "username" pref introduced in https://github.com/mozilla/loop/pull/444/files#diff-705bc24cfa6f5c1421b6e20e28b62660R1495 ?